layouts/shortcodes/observability_pipelines/source_settings/kafka.md

```diff
@@ -11,4 +11,5 @@
     1. Check your values against the [librdkafka documentation][10131] to make sure they have the correct type and are within the set range.
     1. Click **Add Option** to add another librdkafka option.
 
-[10131]: https://github.com/confluentinc/librdkafka/blob/master/CONFIGURATION.md
\ No newline at end of file
+[10131]: https://github.com/confluentinc/librdkafka/blob/master/CONFIGURATION.md
+[10132]: /observability_pipelines/advanced_configurations/
```

Comment:
Is this supposed to be [10172]?

Reply:
No, if a shortcode already had a link list started, then I just used that set of numbers. Thanks for asking!